cpp/include/cudf/utilities/legacy/wrapper_types.hpp

```diff
@@ -340,28 +340,28 @@ using unwrapped_type_t = typename unwrapped_type<T>::type;
 
 } // namespace detail
 
-using category = detail::wrapper<gdf_category, GDF_CATEGORY>;
+using category = detail::wrapper<int32_t, GDF_CATEGORY>;
```

Comment:
Are you sure this is what we should do here? I thought that the wrapper types should map a cuDF type enum value to a cudf type. I think it's clearer if that cudf type is an alias rather than a std type.

Reply:
I wasn't sure about this. Jake had suggested using the type directly here and removing the gdf_ typedefs altogether, but I didn't end up removing them. Is it correct that these wrapper types are going to end up getting replaced in the transition to cudf++? Maybe I should just leave these alone then.

Reply:
`cudf::category` _is_ the alias. The fact that it wraps an `int32_t` is an implementation detail. The previous `gdf_category` was created before the `cudf::category` existed. Now that `cudf::category` exists, there's no reason to use a type alias for the underlying type of the wrapper.